tests/resources/technical-records.json

```diff
@@ -40371,115 +40371,343 @@
     ]
   },
   {
-    "systemNumber": "5917413",
```

Comment:
always worried when changing the seed data it's going to break regression packs. I assume these are records we're added?

Reply:
I don't exactly know what happened to that file, but all I did was add a new trailer to the end and it seems to have messed everything up, I'll try and clean it.

Reply:
https://github.com/dvsa/cvs-svc-technical-records/pull/269 just got merged into develop which also added seed data, which may be the reason why as it will need rebasing.